### PR TITLE
Add CI job for Azure based multi-arch compute cluster

### PR DESCRIPTION
**Pull request by SaravanaStorageNetwork**
CI job for Azure based multi-arch compute cluster test runs
/cc @aleskandro @jeffdyoung

### CODE REVIEW

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.13.yaml [hunk 1]
```diff
@@ -489,6 +489,31 @@ tests:
       BRANCH: "4.13"
       TEST_TYPE: conformance-serial
     workflow: openshift-e2e-powervs-ipi
+- as: ocp-e2e-azure-ovn-heterogeneous
```
Comment (aleskandro) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.13.yaml:492]: You missed a make update before committing

Comment (SaravanaStorageNetwork) [reply ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.13.yaml:492]: Ack @aleskandro - Also, I will update for 4.14 release as well.

**Comment by aleskandro** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.13-ocp-e2e-azure-ovn-heterogeneous

**Review by deepsm007 — APPROVED** [review]

**Review by bradmwilliams — APPROVED** [review]

**Comment by SaravanaStorageNetwork** [comment]
Few tests were failing..Now fixed. 
PTAL 

/cc @deepsm007  @bradmwilliams

**Review by deepsm007 — APPROVED** [review]
/lgtm

**Pull request merged**